src/lib/metric_ingest.py

```diff
@@ -59,7 +59,7 @@ async def _push_to_dynatrace(context: Context, lines_batch: List[IngestLine]):
         context.log("Ingest input is: ")
         context.log(ingest_input)
     ingest_response = await context.session.post(
-        url=urljoin(context.dynatrace_url, "/api/v2/metrics/ingest"),
+        url=f"{context.dynatrace_url}/api/v2/metrics/ingest",
```

Comment:
If we want to do a simple string appending, then we need to handle double slash case - if `context.dynatrace_url` ends with `/`, then result path has empty URL segment and Dynatrace server returns 404.

Something like that should work
```suggestion
        url=f"{context.dynatrace_url.rstrip('/')}/api/v2/metrics/ingest",
```